code/modules/mob/living/bloodcrawl.dm

```diff
@@ -91,7 +91,7 @@
 	return TRUE
 
 /mob/living/proc/bloodcrawl_consume(mob/living/victim)
-	to_chat(src, "<span class='danger'>You begin to feast on [victim]. You can not move while you are doing this.</span>")
+	to_chat(src, "<span class='danger'>You begin to feast on [victim]... You can not move while you are doing this.</span>")
```

Comment:
This looks odd. Can you even use ``...`` in this way?

Reply:
There's multiple situations where there's an ellipse used when telling the user they're currently doing something which needs them to hold still. I quite like it and I'd like it even better if there were less hand-holding and it would just end with "You begin to feast on [victim]..." and the user would realize to hold still (plus from seeing the do_after-bar above them probably).

Reply:
As far as I am aware ``...`` does not start a new sentence so the ``You`` after it shouldn't be capitalized.
Could be wrong on that though.